components.go

```diff
@@ -92,3 +94,8 @@ func (e *Event) SetStatus(status EventStatus) {
 		e.Props.SetText(PropStatus, string(status))
 	}
 }
+
+// Returns the Recurrence Set for this event.
+func (e *Event) RecurrenceSet(loc *time.Location) (*rrule.Set, error) {
+	return e.Props.RecurrenceSet(loc)
+}
```

Comment:
Since RRULE is not specified to events (can also be used by todos etc), I think I'd rather let users call `Props.RecurrenceSet` directly.

Reply:
How about putting it in `Component`? It seems it can be part of `VEVENT`, `VTODO` and `VJOURNAL`, right?

Reply:
Looks fine to me. No need to have it in `Props` then.